documentation/src/main/asciidoc/topics/con_invalidation_caches.adoc

```diff
@@ -17,23 +24,23 @@ endif::downstream[]
 Sometimes the application reads a value from the external store and wants to write it to the local cache, without removing it from the other nodes.
 To do this, it must call `Cache.putForExternalRead(key, value)` instead of `Cache.put(key, value)`.
 
-Invalidation mode can be used with a shared cache store.
-A write operation will both update the shared store, and it would remove the stale values from the other nodes' memory.
-The benefit of this is twofold: network traffic is minimized as invalidation messages are very small compared to replicating the entire value, and also other caches in the cluster look up modified data in a lazy manner, only when needed.
+When you use invalidation mode with a shared cache store, write operations both update the shared store, and remove the stale values from the other nodes' memory.
+The benefit of this is twofold: minimalizing network traffic, as invalidation messages are very small compared to replicating the entire value, and also other caches in the cluster look up modified data in a lazy manner, only when needed.
 
 [IMPORTANT]
 ====
+Invalidation mode is suitable only for shared stores where all nodes can access the same data.
 Never use invalidation mode with a local, non-shared, cache store.
 The invalidation message will not remove entries in the local store, and some nodes will keep seeing the stale value.
 ====
 
 An invalidation cache can also be configured with a special cache loader, `ClusterLoader`.
 When `ClusterLoader` is enabled, read operations that do not find the key on the local node will request it from all the other nodes first, and store it in memory locally.
-In certain situation it will store stale values, so only use it if you have a high tolerance for stale values.
+This can lead to storing stale values, so only use it if you have a high tolerance for stale values.
```

Comment:
This sounds like a bug that should be remedied.. It would be simple to mark a key as being looked up and then not cache the value if an invalidation occurs in the middle of it.

Reply:
Not sure how to approach this. I can create an issue but it needs some investigation. 
The docs can be updated later